File: premio.java
import java.util.Scanner;

public class premio {
    public static char verificaPremio(int somaTotal) {
        if (somaTotal >= 150) return 'B';
        if (somaTotal >= 120) return 'D';
        if (somaTotal >= 100) return 'P';
        return 'N';
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int numPao, numDoce, numBolo, somaTotal;


        numPao = scanner.nextInt();
        numDoce = scanner.nextInt();
        numBolo = scanner.nextInt();


        somaTotal = numPao * 1 + numDoce * 2 + numBolo * 3;


        System.out.println(verificaPremio(somaTotal));

        scanner.close();
    }
}

File: premio.py
def verifica_premio(soma_total):
    if soma_total >= 150:
        return 'B'
    if soma_total >= 120:
        return 'D'
    if soma_total >= 100:
        return 'P'
    return 'N'

def main():
    
    num_pao = int(input())
    num_doce = int(input())
    num_bolo = int(input())

    
    soma_total = num_pao * 1 + num_doce * 2 + num_bolo * 3

    
    print(verifica_premio(soma_total))

if __name__ == "__main__":
    main()
